input/tail.md

```diff
@@ -218,6 +218,18 @@ The number of lines to read with each I/O operation.
 
 If you see `chunk bytes limit exceeds for an emitted event stream` or similar log with `in_tail`, set a smaller value.
 
+### `read_bytes_limit_per_second`
+
+| type    | default        | version |
+|:--------|:---------------|:--------|
+| size    | -1 (unlimited) | 1.12.0  |
+
+The number of reading bytes per second to read with I/O operation.
+
+If you work with a big cluster with high volume of log, you can use this parameter
+to avoid network saturation and make it easier to calculate the max throughput per node.
+This option is useful. To restrict shipping log volumes per second, set a positive number.
```

Comment:
> `This option is useful`

This sentence isn't needed.

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/259/commits/08df02e330b8406f7332e7bfb307facfda7b7687.